src/sentry/eventstream/kafka/backend.py

```diff
@@ -28,7 +28,17 @@ def delivery_callback(self, error, message):
         if error is not None:
             logger.warning("Could not publish message (error: %s): %r", error, message)
 
-    def _send(self, project_id, _type, extra_data=(), asynchronous=True):
+    def _send(
+        self,
+        project_id,
+        _type,
+        extra_data=(),
+        asynchronous=True,
+        headers=None,  # type: Optional[Mapping[str, str]]
```

Comment:
This won't actually type check with `mypy` because `typing` is not imported, but these should be valid type comments per [PEP 484](https://www.python.org/dev/peps/pep-0484/). Regardless of whether or not we ever type check in this repository, they are just useful as documentation.

Reply:
```
./src/sentry/eventstream/snuba.py:223:24: F821 undefined name 'Optional'
./src/sentry/eventstream/snuba.py:223:24: F821 undefined name 'Mapping'
./src/sentry/eventstream/snuba.py:235:24: F821 undefined name 'Optional'
./src/sentry/eventstream/snuba.py:235:24: F821 undefined name 'Mapping'
./src/sentry/eventstream/kafka/backend.py:37:24: F821 undefined name 'Optional'
./src/sentry/eventstream/kafka/backend.py:37:24: F821 undefined name 'Mapping'
```

🙃